mm2src/mm2_main/src/lp_swap/taker_swap.rs

```diff
@@ -204,6 +208,8 @@ impl TakerSavedEvent {
 #[derive(Debug, Deserialize, PartialEq, Serialize)]
 pub struct TakerSavedSwap {
     pub uuid: Uuid,
+    #[cfg(all(not(target_arch = "wasm32"), feature = "new-db-arch"))]
+    pub maker_address: String,
```

Comment:
Are we sure this will not break any backward compatibility when activated? I mean deserialization of already saved swaps. I think there are tests covering this case and we will see when we activate it how it's handled.

Reply:
it surely will. i am currently not thinking about the transition and migration things. will do this collectively at the end (hopefully) as thinking about that right now just slows down the devving so bad.

Reply:
Good plan :)